src/transformers/models/t5/modeling_flax_t5.py

```diff
@@ -1376,15 +1427,20 @@ def setup(self):
         encoder_config.is_decoder = False
         encoder_config.is_encoder_decoder = False
         encoder_config.causal = False
-        self.encoder = FlaxT5Stack(encoder_config, embed_tokens=self.shared, dtype=self.dtype)
+        self.encoder = FlaxT5Stack(
+            encoder_config,
+            embed_tokens=self.shared,
+            dtype=self.dtype,
+            gradient_checkpointing=self.gradient_checkpointing,
+        )
 
     def __call__(
         self,
         input_ids=None,
         attention_mask=None,
         output_attentions=False,
         output_hidden_states=False,
-        return_dict=True,
+        return_dict: bool = True,
```

Comment:
And again! Not necessary once the `return_dict` kwarg is removed from the `layer_module` call in `FlaxT5BlockCollection`